web/src/constant.ts

```diff
@@ -2,7 +2,9 @@ export const PROVIDERS = {
   GITHUB: 'github',
   TWITTER: 'twitter',
   YOUTUBE: 'youtube',
+  GOOGLE: 'google',
   YOUTUBE_READ_ONLY_SCOPE: 'https://www.googleapis.com/auth/youtube.readonly',
   YOUTUBE_CHANNELS_API: 'https://www.googleapis.com/youtube/v3/channels?part=snippet&mySubscribers=true',
-  TRIBE_COUNT_API: '' // place the cloud function url
+  TRIBE_COUNT_API: '', // @TODO place the cloud function url
+  AUTHORIZATION_PASSWORD: 'TribeDashboard'
```

Comment:
If this is in the UI it is not secure

Reply:
As mentioned before on previous PR when discussing the pending `@TODO` on this topic, it would be easier and better to use the `callable` function via the SDK that provides info to the function if the user is authenticated or not, firebase docs here https://firebase.google.com/docs/functions/callable

Reply:
@eddiejaoude The above functionality can be implemented only when the user is logged In. Since we are using the cloud function for homepage the users will not be logged In so the context data in the callable function would be null. So we cannot authenticate using the above method. I think it would be better if we enable CORS for the cloud functions. Let me know your thoughts on this ?

Reply:
hmm **CORS** is only for the web, it doesn't protect against another server. **CORS** would be better than managing our own custom token

Reply:
@eddiejaoude So shall I proceed with CORS implementation.

Reply:
Yes I think it would be better than the custom token

Reply:
Actually why do we want to do it as a http, why not as a normal sdk get data?

Reply:
@eddiejaoude Thought of keeping the functionality like an API which returns the list of top tribe users.

Reply:
I think everything else like this is via the SDK, I would keep it the same and simple

Reply:
Okay @eddiejaoude Will change it 👍